oxide-auth/src/code_grant/authorization.rs

```diff
@@ -170,7 +172,7 @@ impl Authorization {
     }
 
     /// Go to next state
-    pub fn advance<'req>(&mut self, input: Input<'req>) -> Output<'_> {
+    pub fn advance(&mut self, input: Input) -> Output {
```

Comment:
I don't actually know for certain if this is breaking. At face value it looks like this is actually exactly the same. [Reference](https://doc.rust-lang.org/stable/reference/lifetime-elision.html)

- Each elided lifetime in the parameters becomes a distinct lifetime parameter.
- If the receiver has type &Self or &mut Self, then the lifetime of that reference to Self is assigned to all elided output lifetime parameters.